easybuild/easyconfigs/m/MOPAC/MOPAC-23.1.2-gfbf-2024a.eb

```
@@ -0,0 +1,29 @@
+easyblock = 'CMakeMake'
+name = 'MOPAC'
+version = '23.1.2'
+
+homepage = 'https://github.com/openmopac/mopac'
+description = """MOPAC is a general-purpose semiempirical molecular orbital package
+for the study of solid state and molecular structures and reactions."""
+
+toolchain = {'name': 'gfbf', 'version': '2024a'}
+toolchainopts = {'openmp': True}
+
+builddependencies = [
+    ('CMake', '3.29.3'),
+    # required by test suite
+    ('SciPy-bundle', '2024.05'),
+]
+
+source_urls = ['https://github.com/openmopac/mopac/archive']
+sources = ["v%(version)s.tar.gz"]
+checksums = ['60436bbf62045f06f17b4604bf241c8c6359a70a89c41d00913833bf32ea0121']
+
+sanity_check_paths = {
+    'files': ['bin/mopac', 'lib/libmopac.so', 'include/mopac.h'],
+    'dirs': ['bin', 'lib', 'include'],
+}
+
```

please add sanity_check_commands?
```suggestion

sanity_check_commands = [
    'echo | mopac',
    'echo | mopac-makpol',
    'echo | mopac-param',
]

```